xls/simulation/cocotb/cocotb_xls.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2023 The XLS Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import cocotb
+
+from cocotb.triggers import RisingEdge
+from cocotb.binary import BinaryValue
+from cocotb.handle import SimHandleBase
+
+from cocotb_bus.drivers import BusDriver
+from cocotb_bus.monitors import BusMonitor
+
+from typing import Any, List
+
+
+class XLSChannelDriver(BusDriver):
+    _signals = ["data", "rdy", "vld"]
+
+    def __init__(
+        self, entity: SimHandleBase, name: str, clock: SimHandleBase, **kwargs: Any
+    ):
+        BusDriver.__init__(self, entity, name, clock, **kwargs)
+
+        self.bus.data.setimmediatevalue(0)
+        self.bus.vld.setimmediatevalue(0)
+
+    async def write(self, data: List[BinaryValue], sync: bool = True) -> None:
```

Comment:
shouldn't we implement https://github.com/cocotb/cocotb-bus/blob/master/src/cocotb_bus/drivers/__init__.py#L149C15-L149C27 here instead of defining a new interface?

Reply:
implemented methods required for the `send()` function and removed the `write()`